src/modules/Contribute/pages/service.tsx

```diff
@@ -23,7 +23,7 @@ const EMAIL_SUPPORT = 'contribute@opentermsarchive.org';
 
 const ServicePage = ({ documentTypes }: { documentTypes: string[] }) => {
   const router = useRouter();
-  const { t } = useTranslation();
+  const { t } = useTranslation(['contribute/service', 'contribute']);
```

Comment:
I think we will end up with a lot of different files and I'm wondering if we really need this.
Considering the size of the website, everything that is in the `Contribute` module should be in the namespace `contribute`

TBD

Reply:
The `contribute` module have some subpages and as we would like one translation file (`.json`) per page (`.tsx`) we need to configure a namespace for them.